modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -196,41 +196,51 @@ RadialReturnStressUpdateTempl<false>::updateState(RankTwoTensor & strain_increme
   // compute the effective trial stress
   Real dev_trial_stress_squared =
       deviatoric_trial_stress.doubleContraction(deviatoric_trial_stress);
-  Real effective_trial_stress = std::sqrt(3.0 / 2.0 * dev_trial_stress_squared);
 
-  // Set the value of 3 * shear modulus for use as a reference residual value
-  _three_shear_modulus = 3.0 * ElasticityTensorTools::getIsotropicShearModulus(elasticity_tensor);
-
-  computeStressInitialize(effective_trial_stress, elasticity_tensor);
-
-  // Use Newton iteration to determine the scalar effective inelastic strain increment
-  _scalar_effective_inelastic_strain = 0.0;
-  if (!MooseUtils::absoluteFuzzyEqual(effective_trial_stress, 0.0))
+  if (MooseUtils::absoluteFuzzyEqual(dev_trial_stress_squared, 0.0))
   {
-    returnMappingSolve(effective_trial_stress, _scalar_effective_inelastic_strain, _console);
-    if (_scalar_effective_inelastic_strain != 0.0)
-      inelastic_strain_increment =
-          deviatoric_trial_stress *
-          (1.5 * _scalar_effective_inelastic_strain / effective_trial_stress);
-    else
-      inelastic_strain_increment.zero();
+    inelastic_strain_increment.zero();
```

Comment:
This code would be skipping `computeStressInitialize()` in this case. I'm not sure if that's important, but it would be different from the current behavior.

Reply:
yeah, better be safe and call it.